internal/pkg/connectors/wise/task_fetch_transfers.go

```diff
@@ -57,7 +57,7 @@ func matchTransferStatus(status string) payments.Status {
 		return payments.StatusSucceeded
 	case "bounced_back", "funds_refunded":
 		return payments.StatusFailed
-	case "cancelled":
+	case "canceled":
```

Comment:
Are you sure of this change?
It looks like a `misspell` change, but both spellings are correct